tests/ArmTemplates.Tests/Common/ExtensionsTest.cs

```diff
@@ -0,0 +1,42 @@
+﻿// --------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+// --------------------------------------------------------------------------
+
+using Xunit;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Extensions;
+using FluentAssertions;
+
+namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Tests.Common
+{
+    public class ExtensionsTest
```

Comment:
We categorize tests with `trait` - [Organizing tests with Traits in XUnit](http://www.brendanconnolly.net/organizing-tests-with-xunit-traits/) (take a look at extractor scenarios tests). Personally I never use them, but maybe at some time it will be usefull.

We can either **remove them completely** or **add them here also**. What do you think?

Reply:
This applies to other tests as well

Reply:
Agree added to introduced and modified tests